screenshot.html

```diff
@@ -248,8 +248,14 @@ <h3 id=screenshot-header></h3>
       toggleElements(false);
       const screenshotResult = document.getElementById('screenshot-result');
 
-      let {url, hash, ext, type, dst} = parseQuery(decodeURIComponent(window.location.search));
-
+      let {url, hash, ext, type, dst, width = 512, height = 512} = parseQuery(decodeURIComponent(window.location.search));
+      // make sure the image is square
+      let aspectRatio = width / height;
```

Comment:
We want the exact size requested to come back. We shouldn't change the result dimensions.

The renders just need to fit in the requested dimension.